CONTRIBUTING.md

```diff
@@ -177,31 +177,26 @@ which shows the baseline image, the generated image, the diff and the json mocks
 
 To view the results of a run on CircleCI, download the `build/test_images/` and `build/test_images_diff/` artifacts into your local repo and then run `npm run start-image_viewer`.
 
-### Writing selection tests
-Keep in mind that the selection coordinates are relative to the top-left corner of the plot, including the margins. To produce a reliable selection test, 
+### Writing interaction tests
+Keep in mind that the interaction coordinates are relative to the top-left corner of the plot, including the margins. To produce a reliable interaction test, 
 it may be necessary to to fix the width, height, margins, X axis range and Y axis range of the plot. For example:
 
 ```
-Plotly.newPlot(gd,
-    [
-        {
+Plotly.newPlot(gd, [{
             x: [1, 1, 1, 2, 2, 2, 3, 3, 3],
             y: [1, 2, 3, 1, 2, 3, 1, 2, 3],
             mode: 'markers'
-        }
-    ],
-    {
-        width: 400, height: 400,
-        margin: {l: 100, r: 100, t: 100, b: 100},
-        xaxis: {range: [0, 4]},
-        yaxis: {range: [0, 4]}
-     }
-)
+        }], {
+            width: 400, height: 400,
+            margin: {l: 100, r: 100, t: 100, b: 100},
+            xaxis: {range: [0, 4]},
+            yaxis: {range: [0, 4]}
+     });
```

Comment:
Hmm. That's not quite right. Make sure there are 4 spaces before the keys and none before the `}], {` and `]);` lines.